From 25175799d94b7fa597bb0f8060f7213242a53ceb Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sat, 31 Oct 2020 08:34:01 +0100
Subject: [PATCH 1/1] test: test/bloblist.c depends on asm/state.h

Building test/bloblist.c fails for non sandbox devices:

test/bloblist.c:10:10: fatal error: asm/state.h: No such file or directory
 #include <asm/state.h>
          ^~~~~~~~~~~~~

Build the test only on the sandbox.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 test/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/test/Makefile b/test/Makefile
index 1c930b3148..39ae04a3d2 100644
--- a/test/Makefile
+++ b/test/Makefile
@@ -2,7 +2,9 @@
 #
 # (C) Copyright 2012 The Chromium Authors
 
+ifneq ($(CONFIG_SANDBOX),)
 obj-$(CONFIG_$(SPL_)CMDLINE) += bloblist.o
+endif
 obj-$(CONFIG_$(SPL_)CMDLINE) += cmd/
 obj-$(CONFIG_$(SPL_)CMDLINE) += cmd_ut.o
 obj-$(CONFIG_$(SPL_)CMDLINE) += command_ut.o
-- 
2.20.1

